docker-compose.abuse.yml

```diff
@@ -0,0 +1,38 @@
+version: "3.7"
+
+x-logging: &default-logging
+  driver: json-file
+  options:
+    max-size: "10m"
+    max-file: "3"
+
+services:
+  abuse:
+    build:
+      context: ./docker/abuse
+      dockerfile: Dockerfile
+    container_name: abuse
+    restart: unless-stopped
+    logging: *default-logging
+    env_file:
+      - .env
+    environment:
+      - ABUSE_LOG_LEVEL=${ABUSE_LOG_LEVEL}
+      - ABUSE_MAILBOX=${ABUSE_MAILBOX}
+      - ABUSE_SPONSOR=${ABUSE_SPONSOR}
+      - BLOCKER_IP=${BLOCKER_IP}
+      - BLOCKER_PORT=${BLOCKER_PORT}
```

Comment:
`abuse` is now using `BLOCKER_HOST` and not `BLOCKER_IP`. This is not a real problem, though, because you have passed the `.env` file, so all values from it will be available to the service, whether they are deifned here or not.

We can also add those values here already, like this:
```
      - SKYNET_DB_HOST=${SKYNET_DB_HOST:-mongo}
      - SKYNET_DB_PORT=${SKYNET_DB_PORT:-27017}
```